### DISCUSSION TITLE
2022-10-22-weekly

### DISCUSSION THREAD

**Discussion by tw93** [tw93/tw93.github.io#107] [General]
https://tw93.fun/2022-10-22/weekly.html 

 Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Life之前潮流周刊放到了 Github 上，不是太好用，趁着国庆那几天，学习了下 Astro ，自己设计开发了一个潮流周刊的官网，终于感觉舒服一些了，分享出来给大家。

**Comment by xiaowuleyi** [comment]
非常优秀，看起来整体非常简洁优雅。

**Reply by tw93** [reply]
谢谢喜欢

**Comment by tw93** [comment]
> @xiaowuleyi
> 非常优秀，看起来整体非常简洁优雅。

谢谢喜欢

**Comment by dxhuii** [comment]
RSS 用的是什么软件？

**Reply by tw93** [reply]
https://netnewswire.com/
比较好用简洁，推荐

**Reply by dxhuii** [reply]
谢谢。

**Comment by weiqifa0** [comment]
很酷，我是做嵌入式的，超级喜欢你做的东西。

**Reply by tw93** [reply]
谢谢喜欢

**Comment by Liqishun** [comment]
hi,我想问下这个网站的技术栈。

**Reply by tw93** [reply]
可见源码 https://github.com/tw93/tw93.github.io

**Comment by venomwise** [comment]
泰裤辣

**Comment by MrGodfrey** [comment]
非常棒！
